charts/datahub/templates/datahub-upgrade/datahub-restore-indices-job-template.yml

```diff
@@ -12,6 +12,7 @@ metadata:
 spec:
   schedule: "* * * * *"
   suspend: true
+  concurrencyPolicy: {{ .Values.datahubUpgrade.cleanupJob.concurrencyPolicy | default "Allow" }}
```

Comment:
This should have been `.Values.datahubUpgrade.restoreIndices.concurrencyPolicy`. See the README file and add an update it [here](https://github.com/acryldata/datahub-helm/blob/ae3db71b53e08731e54bf21a954dd969ceece677/charts/datahub/README.md?plain=1#L43)

Reply:
Done.

Reply:
thanks for the catch